_includes/manuals/1.16/3.5.2_configuration_options.md

```diff
@@ -83,68 +83,9 @@ Some plugins may add their own loggers. See the configuration files in /etc/fore
 
 #### The 'email.yaml' file
 
-This settings file can be found at `/etc/foreman/email.yaml`, or `config/email.yaml` on a source installation.
+**This file has been deprecated for configuring email, SMTP and sendmail options. Please remove it and configure via the web UI under *Administer > Settings > Email* instead. On Foreman 1.16 or later, this file will be completely useless.**
```

Comment:
I think this should be in the Upgrade guide and release notes. The reference manual shouldn't document the outdated section.